httpx/_dispatch/wsgi.py

```diff
@@ -7,6 +8,19 @@
 from .base import SyncDispatcher
 
 
+def _get_non_empty_chunk(body: typing.Iterable) -> typing.Iterable:
```

Comment:
Would this be a more appropriate name for what this does, i.e. skip any empty chunks in the `body`?

```suggestion
def _seek_non_empty_chunk(body: typing.Iterable) -> typing.Iterable:
```